modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.tsx

```diff
@@ -82,6 +82,7 @@ const Card = ({item, schema}: {item: any; schema: ICardSchema}) => {
 			description={localizedDescription}
 			href={(schema.link && item[schema.link]) || null}
 			imgProps={imageProps}
+			labels={schema.labels}
```

Comment:
Hey @NemethNorbert, changes look fine to me. I'm curious about this mapping though. Here you set labels directly as they come from the schema. This means that label values are not being taken from the `item` but are a fixed value set by the card schema, the same for all rendered cards.

This is a very uncommon scenario, though possible in the end.

Is this intended?

Reply:
I am not sure @dsanz, I'd like the labels to be determined by the given item. I might misunderstood what should come from the schema.

Should the `schema.labels` return a string that can be used as a key to retrieve the labels from the item object?